agent/tool-scripts/perf

```diff
@@ -191,14 +191,19 @@ case "$mode" in
 			popd > /dev/null 2>&1
 			# Generate the initial report.
 			$tool_bin $report_opts -i ${tool_output_dir}/perf.data --stdio > ${tool_output_dir}/perf-report.txt 2> ${tool_output_dir}/perf-report.err
-			xz --threads=0 ${tool_output_dir}/perf.data
 			# only postprocess the perf report if the default options are used
+			#create flamegraph if --callgraph option is used
 			if echo $record_opts | grep -E -q -- -g\|--callgraph; then
-				debug_log "$script_name: not going to run perf postprocess script since this report contains a callgraph"
+                perf script -i ${tool_output_dir}/perf.data > ${tool_output_dir}/out.perf
+                /opt/pbench-agent/tool-scripts/stackcollapse-perf.pl ${tool_output_dir}/out.perf > ${tool_output_dir}/out.folded
```

Comment:
IMO, the paths shouldn't be hard-coded but generated similar to script_path

Reply:
You can use `${pbench_bin}/tool-scripts` instead of hard-coding `/opt/pbench-agent`.

Reply:
We have considered that and started from there, but @Anisha1234 and me where facing an issues with path. On placing the code segment in perf-postprocess file, the path of perf data files seems to be appended with hostname and for us the path is the original one in `perf` script which gives us a "No such file or directory". 

eg:
If done in, perf script: `/sample1/tools-default/perf/perf.data`
if done in, perf-postprocess: `/sample1/tools-default/<hostname>/perf/perf.data`

I couldn't spend much time on it but @Anisha1234 took it up and probably discussed with @ndokos .

@Anisha1234 Were you able to figure the problem with the path?